staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -239,7 +250,13 @@ func describerMap(clientConfig *rest.Config) (map[schema.GroupKind]ResourceDescr
 // DescriberFor returns the default describe functions for each of the standard
 // Kubernetes types.
 func DescriberFor(kind schema.GroupKind, clientConfig *rest.Config) (ResourceDescriber, bool) {
-	describers, err := describerMap(clientConfig)
+	return DescriberForClient(kind, clientConfig, nil)
```

Comment:
any callers to this call site will still spawn new transports, right? do we have any callers to this site and can we eliminate them?